### PR TITLE
Updates to the welcome message for new members

### PR DESCRIPTION
**Pull request by joshmerritt**
Randomly selects an icebreaker whenever a new member appears.

### CODE REVIEW

File: stubs/enterReplies.js [hunk 1]
```diff
@@ -11,10 +11,17 @@ icebreakers = [
...
+  'Share your favorite color (in hexcode!)',
+  'What website do you suddenly find yourself on when you\'re \"just checking your email\"?',
+  'If you could be any other species, which would you choose?',
+  'What superpower would you want?',
+  'What\'s your patronus?'
 ];
 
 // Combine introduction with an icebreaker question
-responses = icebreakers.map((item) => introduction + ' ' + item) + 'Start a thread with your response to get to know the hack lab better';
+responses = icebreakers.map((item) => introduction + ' ' + item + 'Start a thread with your response to get to know the hack lab better');
```
Comment (connerk) [inline stubs/enterReplies.js]: I'm not sure what `item` returns but should there be a space in there? `...item + ' Start...`

File: stubs/enterReplies.js [hunk 2]
```diff
@@ -1 +1,27 @@
...
+  'How old were you when you stopped believing in Santa Claus?',
+  'Share your favorite color (in hexcode!)',
+  'What website do you suddenly find yourself on when you\'re \"just checking your email\"?',
+  'If you could be any other species, which would you choose?',
+  'What superpower would you want?',
+  'What\'s your patronus?'
+];
+
+// Combine introduction with an icebreaker question
+responses = icebreakers.map((item) => introduction + ' ' + item + 'Start a thread with your response to get to know the hack lab better');
```
Comment (pcunning) [inline stubs/enterReplies.js]: This approach is nice and simple however these are used for enter/exit from all rooms hackbot is in. If you change that to filter to only general I love it. Also we could make hackbot do the introduction then start a thread on the message it just posted with the comment. Would likely need to @ the user.

File: stubs/enterReplies.js [hunk 3]
```diff
@@ -1 +1,27 @@
...
\ No newline at end of file
+introduction = 'Please join hackbot in welcoming @user to the Rogue Hack Lab!!! :tada:';
+
+icebreakers = [
+  'What\'s your favorite 1990\'s cartoon?',
+  'What did you have for breakfast?',
+  'What\'s one skill you would like to develop?',
+  'What\'s your favorite letter of the alphabet and why?',
+  'How old were you when you first used a computer?',
+  'What\'s your favorite spot for a beer?',
```
Comment (pcunning) [inline stubs/enterReplies.js]: Let’s change this to be a little more age inclusive as we have some <21 people join.

File: stubs/enterReplies.js [hunk 4]
```diff
@@ -1 +1,27 @@
...
+  'How did you hear about the hack lab?',
+  'When was the last time you changed your hairstyle?',
+  'Have you committed today?',
+  'You\'re trapped on a deserted island, what TV show are you bringing to keep you company?',
+  'What\'s your favorite flavor of sparkling water?',
+  'Tell us your favorite \'nerdy\' joke',
+  'How old were you when you stopped believing in Santa Claus?',
```
Comment (pcunning) [inline stubs/enterReplies.js]: Wait what?

**Comment by pcunning** [comment]
Fixes #41

File: stubs/enterReplies.js (hunk 2 above)
Comment (pcunning) [reply stubs/enterReplies.js]: Thinking about this you might need to get running with a test bot to get the right way to tell the channel. Let me know if you want to pair on that.

File: stubs/enterReplies.js (hunk 1 above)
Comment (joshmerritt) [reply stubs/enterReplies.js]: Thanks Kevin! Yes, there does probably need to be a space or newline in there. I'll work with Pat getting the test bot going so I can see the output and get it fined tuned.

File: stubs/enterReplies.js (hunk 2 above)
Comment (joshmerritt) [reply stubs/enterReplies.js]: Thanks Pat! I'd love to get a test bot up and running and be able to do some more thorough testing. I'll read up on how to narrow down the messages. My initial feeling is to just filter the messages in this function, and use some of the older style short messages for other room arrivals. I don't think I've seen many room exits, so I'll need to dig into that more.

File: stubs/enterReplies.js (hunk 3 above)
Comment (joshmerritt) [reply stubs/enterReplies.js]: Definitely, that's a good point!

File: stubs/enterReplies.js (hunk 4 above)
Comment (joshmerritt) [reply stubs/enterReplies.js]: My bad, I should have prefaced this with LIFE SPOILER ALERT.

Comment (xtagon) [reply stubs/enterReplies.js]: Sparkling water, a nerd, and Santa Clause walk into a bar...

**Review by pcunning — APPROVED** [review]

**Pull request merged**